adafruit_vcnl4040.py

```diff
@@ -38,6 +38,12 @@
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_VCNL4040.git"
 
 
+try:
+    from busio import I2C
+except ImportError:
+    pass
```

Comment:
For the import here we can try importing typing first like this:
```
try:
    import typing
    from busio import I2C
except ImportError:
    pass`
```
the `typing` module doesn't exist on microcontrollers so this will fail to import and then prevent it from importing `busio` (or anything else that we would need strictly for typing rather than functionality). Which will save a bit of RAM space for us

Reply:
Thanks for the detailed answer! When I added `import typing` in the try except block, the pre-commit test fails at `pylint (library code)`. More specifically, the pylint error is `adafruit_vcnl4040.py:42:4: W0611: Unused import typing (unused-import)`. Should I commit the changes anyway?

Reply:
Yup, typically `pylint` would be correct, but the behavior @FoamyGuy detailed should override that.  You can disable that specific warning using a comment:

```python
try:
    import typing  # pylint: disable=unused-import
    from busio import I2C
except ImportError:
    pass
```

That'll let `pylint` know you understand it's anger but don't care 😆